### DISCUSSION TITLE
Export entire DB from front-end with button

### DISCUSSION THREAD

**Discussion by aidenmitchell** [Budibase/budibase#6834] [Help] [answered]
Is there a way to export an entire DB from the front-end (button click), rather than having to export from the back-end?
So far, users can export selected rows with a button, but we'd like to export the whole DB (or at least 500 rows at a time.)

**Comment by melohagan** [comment, marked as answer]
Hey @aidenmitchell 

At the moment exporting a table only applies to selected rows. What you could try is an automation with looping: https://docs.budibase.com/docs/automation-actions#looping
Then you could send an email with the table rows: https://github.com/Budibase/budibase/discussions/6162
